sdk/opendp/smartnoise/sql/private_reader.py

```diff
@@ -32,9 +32,17 @@ def __init__(self, metadata, reader, epsilon_per_column=1.0, delta=10E-16,
             :param options: A PrivateReaderOptions with flags that change the behavior of the privacy
                 engine.
         """
-        self.options = options if options is not None else PrivateReaderOptions()
-        self.metadata = metadata
-        self.reader = reader
+        if isinstance(reader, Reader):
```

Comment:
I think for the first release of this change we can log a warning if the order is flipped.

if type(metadata, Reader) and type(reader, CollectionMetadata):
     log a warning
      tmp = reader
      reader = metadata
      metadata = tmp

This could be added before the current code for now. And removed later